### PR TITLE
Add document list command

### LINKED ISSUE #273: Add admin commands
**Description**:

Add admin commands.

Let's implement admin commands, starting with finding and deleting documents.

```bash
# starts server
$ yorkie server
INFO	default	backend created: id: ca81i5jhkdflkq76edr0, rpc: localhost:11103: db: memory
INFO	default	serving profiling on 11102
INFO	default	serving RPC on 11101
INFO	default	serving admin on 11103

$ yorkie --admin-addr localhost:11103 document describe <doc-key>
$ yorkie --admin-addr localhost:11103 document rm <doc-key>
```

Task
- [x] Create project: `yorkie project create <name>`
- [x] List projects: `yorkie project ls`
- [x] History: `yorkie history <project_name> <document_key>`
- [ ] List documents: `yorkie document ls <project_name>`
- [ ] Remove project: `yorkie project rm <name>`
- [ ] Remove document: `yorkie document rm <project_name> <document_key>`

**Why**:

- Users can manage documents or projects using CLI.

### PR DESCRIPTION
**Pull request by Eithea**
This commit adds document list command.
It can use by `yorkie document ls [ProjectName]`

```
$ ./bin/yorkie document ls default
 ID  KEY  CREATED AT  ACCESSED AT  UPDATED AT  SNAPSHOT 

```

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #273 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Users can view documents using CLI.
```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Comment by Eithea** [comment]
i learnd how to make docs and check the data in local yorkie. I'll make sure it's the right implementation

**Comment by Eithea** [comment]
```
$ ./bin/yorkie document ls default
 ID                        KEY               CREATED AT      ACCESSED AT     UPDATED AT      SNAPSHOT
 62da0c1326a9f40cfb129f86  codepairs$xkz62   8 seconds       2 seconds       2 seconds       {"content":"","shapes":[{"color":"#f3b328","points
 62da0bf726a9f40cfb129c46  codepairs$7sxmw   36 seconds      21 seconds      21 seconds      {"content":"그림도그려져요","shapes":[{"box
 62da0bdf26a9f40cfb129aee  codepairs$nkrwp   About a minute  56 seconds      56 seconds      {"content":"한글이잘나와요","shapes":[]}
 62da0bdb26a9f40cfb1299f9  codepairs$tjmz8   About a minute  About a minute  About a minute  {"content":"안녕하세요\n","shapes":[]}
 62da0b4d4758743aa9341c99  codepairs$85owv   3 minutes       About a minute  About a minute  {"content":"dasdsadsadasdsdsadadsa","shapes":[]}
 62da0b484758743aa9341c04  codepairs$f9gc1   3 minutes       3 minutes       3 minutes       {"content":"dsaddassdsads","shapes":[]}
```

**Comment by hackerwins** [comment]
@Eithea How about adding an ellipsis(`...`) to the snapshot if it is truncated.
https://en.wikipedia.org/wiki/Ellipsis

**Comment by Eithea** [comment]
```
 ID                        KEY               CREATED AT  ACCESSED AT  UPDATED AT  SNAPSHOT
 62da0c1326a9f40cfb129f86  codepairs$xkz62   7 hours     7 hours      7 hours     {"content":"","shapes":[{"color":"#f3b32 ...
 62da0bf726a9f40cfb129c46  codepairs$7sxmw   7 hours     7 hours      7 hours     {"content":"그림도그려져요","shap ...
 62da0bdf26a9f40cfb129aee  codepairs$nkrwp   7 hours     7 hours      7 hours     {"content":"한글이잘나와요","shap ...
 62da0bdb26a9f40cfb1299f9  codepairs$tjmz8   7 hours     7 hours      7 hours     {"content":"안녕하세요\n","shapes": ...
 62da0b4d4758743aa9341c99  codepairs$85owv   7 hours     7 hours      7 hours     {"content":"dasdsadsadasdsdsadadsa","sha ...
 62da0b484758743aa9341c04  codepairs$f9gc1   7 hours     7 hours      7 hours     {"content":"dsaddassdsads","shapes":[]}
 62da0b384758743aa9341ab4  codepairs$fs38ij  7 hours     7 hours      7 hours     {"content":"dsadasds","shapes":[]}
 62da0b2e4758743aa9341a2a  codepairs$tupxoa  7 hours     7 hours      7 hours     {"content":"dsadasdasdas","shapes":[]}
 62da0b2b4758743aa93419cd  codepairs$41mtro  7 hours     7 hours      7 hours     {"content":"dsdsadadas","shapes":[]}
 62da0b294758743aa9341988  codepairs$17ztht  7 hours     7 hours      7 hours     {"content":"dsadasdasdsa","shapes":[]}

14 docs in project default (page 1/2)
j: prev   k: next   q: quit

```

**Comment by Eithea** [comment]
```
 ID                        KEY               CREATED AT  ACCESSED AT  UPDATED AT  SNAPSHOT
 62da0c1326a9f40cfb129f86  codepairs$xkz62   7 hours     7 hours      7 hours     {"content":"","shapes":[{"color":"#f3b32 ...
 62da0bf726a9f40cfb129c46  codepairs$7sxmw   7 hours     7 hours      7 hours     {"content":"그림도그려져요","shap ...
 62da0bdf26a9f40cfb129aee  codepairs$nkrwp   8 hours     7 hours      7 hours     {"content":"한글이잘나와요","shap ...
 62da0bdb26a9f40cfb1299f9  codepairs$tjmz8   8 hours     8 hours      8 hours     {"content":"안녕하세요\n","shapes": ...
 62da0b4d4758743aa9341c99  codepairs$85owv   8 hours     8 hours      8 hours     {"content":"dasdsadsadasdsdsadadsa","sha ...
 62da0b484758743aa9341c04  codepairs$f9gc1   8 hours     8 hours      8 hours     {"content":"dsaddassdsads","shapes":[]}
 62da0b384758743aa9341ab4  codepairs$fs38ij  8 hours     8 hours      8 hours     {"content":"dsadasds","shapes":[]}
 62da0b2e4758743aa9341a2a  codepairs$tupxoa  8 hours     8 hours      8 hours     {"content":"dsadasdasdas","shapes":[]}
 62da0b2b4758743aa93419cd  codepairs$41mtro  8 hours     8 hours      8 hours     {"content":"dsdsadadas","shapes":[]}
 62da0b294758743aa9341988  codepairs$17ztht  8 hours     8 hours      8 hours     {"content":"dsadasdasdsa","shapes":[]}

14 docs in project default (page 1/2)
j: prev                 k: next
a: view all documents   q: quit
```

**Comment by hackerwins** [comment]
The pagination interface looks nice, but I'm wondering if it's NIX style.
https://nixos.org/manual/nix/stable/contributing/cli-guideline.html

Could you leave all logic except for pagination in this PR and separate pagination into another PR?

**Comment by Eithea** [comment]
I reseted the branch to the commit before the pagination.
Pagination should be applied to all CLI equally, I will add issue about this.

File: cmd/yorkie/document/list.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/yorkie-team/yorkie/admin"
+	"github.com/yorkie-team/yorkie/cmd/yorkie/config"
+	"github.com/yorkie-team/yorkie/pkg/units"
+)
+
+func newListCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "ls [project]",
```
Comment (hackerwins) [inline cmd/yorkie/document/list.go:34]:
```suggestion
		Use:   "ls [project name]",
```

**Review by hackerwins — APPROVED** [review]
LGTM. Thanks for your contribution.

**Pull request merged**